distributed/cluster_dump.py

```diff
@@ -198,22 +207,43 @@ def scheduler_story(self, *key_or_stimulus_id: str) -> dict:
 
```

Comment:
If `scheduler_story` is modified to take a `datetime` bool argument, it would be nice if it was set to True in this call.